coder_sniffer/Drupal/Sniffs/Commenting/TodoCommentSniff.php

```diff
@@ -142,7 +142,7 @@ private function checkTodoFormat(File $phpcsFile, int $stackPtr, string $comment
             (?-i)              # Reset to case-sensitive
             (?!                # Start another non-consuming look-ahead, this time negative
               @todo\s          # It has to match lower-case @todo followed by one space
-              (?!-|:)\S        # and then any non-space except "-" or ":".
+              (?!-|:)[A-Z]     # and then any capital letter except "-" or ":".
```

Comment:
this is tricky. We want to warn about lower case characters, but allow everything else. Even lower case characters should be allowed in some cases, for example `@todo some_function() is not a good name, refactor it to a better name`.

I think we have code somewhere else in Coder that checks for such things in some comment standard, we should check that.